services/rds/src/main/java/org/finra/gatekeeper/configuration/GatekeeperProperties.java

```diff
@@ -514,6 +514,12 @@ public GatekeeperProperties setDb(GatekeeperDbProperties gatekeeperDbProperties)
 
```

Comment:
See my comment on GatekeeperEc2Properties about moving this up to the common package